docs/user.md

```diff
@@ -0,0 +1,98 @@
+# User
+
+default response data format: `json`
+default params format of GET/DELETE requests: ```?${params1}=${value1}&${params2}=${values2}```
+default params format of other requests: json in body
+
+## GET /users
+
+* get user info
+
+### Parameters
+
+|Name   |Type   |Optional   |In	Description |
+|-------|-------|-----------|---------------|
+|id     |numbser|t          |user id        |
+|name   |string |t          |user name      |
+
+### Response
```

Comment:
What would the server return, if following errors occur?
- data not found
- internal error (for example: db connection lost)

Reply:
* data not found returns  statuscode 404,
* not handling db connection at all, deamon will keep restarting service. in a higher level, nginx return 500.